setup.py

```diff
@@ -19,7 +19,7 @@
                         "kiwisolver==1.3.1",\
                         "matplotlib==3.3.4",\
                         "networkx==2.5.1",\
-                        "Pillow==9.0.1",\
+                        "Pillow==9.2.0",\
```

Comment:
This version upgrade doesn't change anything detectable by unit tests. It is a fix for a vuln. in previous versions.